Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -96,6 +96,9 @@ public Registry(Uri baseUri)
 
     public async Task<Image?> GetImageManifest(string repositoryName, string reference, string runtimeIdentifier, string runtimeIdentifierGraphPath)
     {
+        if (repositoryName == "scratch") {
```

Comment:
I don't necessarily think that we should try to shove `scratch` into an Image - it almost feels like we need a concept of different 'sources'? This does seem to work, though.